aviatrix/resource_aviatrix_transit_gateway.go

```diff
@@ -851,6 +859,17 @@ func resourceAviatrixTransitGatewayCreate(d *schema.ResourceData, meta interface
 		}
 	}
 
+	var customizeTransitVpcRoute []string
+	for _, v := range d.Get("customize_transit_vpc_route").(*schema.Set).List() {
+		customizeTransitVpcRoute = append(customizeTransitVpcRoute, v.(string))
+	}
+	if len(customizeTransitVpcRoute) != 0 {
+		err := client.UpdateTransitGatewayCustomizedVpcRoute(d.Get("gw_name").(string), customizeTransitVpcRoute)
```

Comment:
Instead of calling `d.Get("gw_name").(string)` again  here we can use `gateway.GwName`